pkg/tsdb/azuremonitor/azure-log-analytics-datasource.go

```diff
@@ -163,6 +163,9 @@ func (e *AzureLogAnalyticsDatasource) executeQuery(ctx context.Context, query *A
 	azlog.Debug("AzureLogAnalytics", "Request ApiURL", req.URL.String())
 	res, err := ctxhttp.Do(ctx, dsInfo.Services[azureLogAnalytics].HTTPClient, req)
 	if err != nil {
+		if !dsInfo.Settings.AzureLogAnalyticsSameAs {
+			return dataResponseErrorWithExecuted(fmt.Errorf("Log Analytics credentials are not being used. Go to the data source configuration to update it. Got error %v", err))
```

Comment:
So if I understand correctly if they get this error it's because they are using Log Analytics credentials when we no longer support Log Analytics sign in right?
In which case I might say "Log Analytics credentials are no longer supported. Go to the data source configuration to update Azure Monitor credentials" and I'm not sure it's really worth it to send along the error message from Azure to them as they can't click the url or really learn more from it.

Reply:
:+1: I like the message. Regarding the URL, I agree that the error message is not helpful, I left it there in case some other error throws a more helpful message but it's likely not worth it to bloat the error that much.